scripts/scale.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+export VALID_NAMESPACES=("hocs-gamma" "hocs-delta" "hocs-qax")
+
+current=$(kubectl config current-context)
+if [[ ${VALID_NAMESPACES[*]} =~ $current ]] ; then
+  echo "Current context is $current"
+else
+  echo "Unauthorised Namespace" >&2
+  exit 1
+fi
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: scale.sh down/up" >&2
+  exit 1
+fi
+
+echo "Please be patient as some of these steps take up to a minute to complete."
+
+if [ "$1" = "down" ]; then
+  kubectl scale --replicas=0 deployment/hocs-management-ui
+  kubectl scale --replicas=0 deployment/hocs-frontend
+  kubectl scale --replicas=0 deployment/hocs-notify
+  kubectl scale --replicas=0 deployment/hocs-search
+  kubectl scale --replicas=0 deployment/hocs-templates
+  sleep 5
```

Comment:
You can probably do something cool with `kubectl wait --for=condition=available` instead of naively sleeping.

Reply:
I agree the wait is the correct way to do this, but we need the pod id (hocs-templates-xxxxxxxxxx-xxxxx). Also what happens if the autoscaler has a minimum of 2 instances set?